less/index.less

```diff
@@ -1,4 +1,23 @@
-// Is this working?
+@primary: black;
+@hover: maroon;
+@top-40: 40px;
+@top-20: 20px;
+
+.flex-100 {
+	width: 100%;
+	display: flex;
+}
+
+.section-block {
+	background-color: lightgrey;
+	width: 100%;
+	padding: 30px;
+	border-radius: 5px;
+	margin-top: @top-40;
+	margin-bottom: @top-40;
+}
+
```

Comment:
nice use of a mixing for making the section block. Now you are able to use that mixin anywhere at anytime.